### PR TITLE
Miscellaneous follow-up improvements to adding Bower support

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/858)

### CODE REVIEW

File: docs/GettingStarted.md [hunk 1]
```diff
@@ -18,23 +18,12 @@ To run ORT the following tools are required:
 * Java (version >= 8)
 * Git
```
Comment (mnonnenmacher) [inline docs/GettingStarted.md]: This should also be removed, as it is now mentioned below.

File: docs/GettingStarted.md [hunk 2]
```diff
@@ -18,23 +18,12 @@ To run ORT the following tools are required:
...
-* [Maven](http://maven.apache.org)
-* [Mercurial](https://www.mercurial-scm.org)
-* [NPM](https://www.npmjs.com) 5.5.* - 6.4.* and [Node.js](https://nodejs.org) 8.* (required for this tutorial because
-  we scan `mime-types` which is an NPM project)
-* [PHP Composer](https://getcomposer.org) >= 1.5
-* [SBT](https://www.scala-sbt.org/)
-* [Stack](https://docs.haskellstack.org/en/stable/README)
-* [Subversion](https://subversion.apache.org)
-* [Yarn](https://yarnpkg.com) 1.3.* - 1.9.*
+For some of the supported package managers and Version Control Systems additional tools need to be installed. In the
```
Comment (mnonnenmacher) [inline docs/GettingStarted.md]: Maybe mention or link the full list of tools in `README.md`.

File: docs/GettingStarted.md (hunk 1 above)
Comment (sschuberth) [reply docs/GettingStarted.md]: Done.

File: docs/GettingStarted.md (hunk 2 above)
Comment (sschuberth) [reply docs/GettingStarted.md]: Done.

**Pull request merged**